### ISSUE TITLE
Add "Give feedback" button to beta

### ISSUE THREAD

**Issue by dasaderi** [PREreview/rapid-prereview#37]
- [x] @dasaderi to make feedback form to link 

- [x] @sballesteros and @halmos to add "Give feedback" button to platform by December 3rd

- [x] add _beta_ somewhere to make it obvious it's a beta. 

Should we have it so that users that try it don't actually end up adding rapid reviews until it's fully launched?

**Comment by dasaderi** [comment]
@sballesteros @halmos and I discussed how this should look like.

I suggested to add a Give Feedback button somewhere on front page linked to a form for users to give anonymous feedback. 

Users will be able to sign in with orcid and complete reviewes as if full launch, but they will we warned that this is still a beta and there might be things that don't work fluently and encouraged to give feedback. @halmos will add a message to reflect that at the end of a rapid review.
